elastic/kibana
# [Infra UI] Asset Details Flyout: Add tooltip with documentation link in metadata summary

## Summary

This PR adds a tooltip with a documentation link in the metadata summary inside the asset details overview section. It should appear only if we have defined documentation link so currently only for the `host.ip` field

<img width="1918" alt="Screenshot 2023-07-18 at 13 13 25" src="https://github.com/elastic/kibana/assets/14139027/e5ef2067-3a3b-48fd-a395-b763a6832bd1">

## Storybook

<img width="1917" alt="image" src="https://github.com/elastic/kibana/assets/14139027/dfbd0118-b0ff-475b-a545-9e7278815098">


## Testing
- Go to host view and open the asset details flyout for any host
- Go to the overview tab
- A question mark icon should appear next to the host ip title inside the metadata section
- Click on the icon and check the tooltip
- the `host.ip` link should open the documentation page for the field in a new tab


https://github.com/elastic/kibana/assets/14139027/c83dcefd-34b1-4f61-aa96-b9fd0e6ef07b
